### DISCUSSION TITLE
KeyError: 'zh_cn'

### DISCUSSION THREAD

**Discussion by yip041154** [xxyzz/WordDumb#112] [General]
May I have anyone help with these error messages? Many thanks. 

calibre, version 6.14.1
錯誤： 未處理的例外: <b>KeyError</b>:'zh_cn'

calibre 6.14.1  embedded-python: True
Windows-10-10.0.19041-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19041')
Python 3.10.1
Windows: ('10', '10.0.19041', 'SP0', 'Multiprocessor Free')
Interface language: zh_TW
Successfully initialized third party plugins: WordDumb (3, 29, 0)
Traceback (most recent call last):
  File "calibre_plugins.worddumb.ui", line 108, in run
  File "calibre_plugins.worddumb.metadata", line 241, in check_word_wise_language
KeyError: 'zh_cn'

**Comment by xxyzz** [comment]
Thanks for your report, https://github.com/xxyzz/WordDumb/commit/46d5971355efc606e993c0b45bef5568352397db fixes this error.
